src/request_handler.rs

```diff
@@ -38,6 +38,14 @@ impl RequestHandler {
     }
   }
 
+  async fn response(&self, request: Request<Body>) -> Result<Response<Body>> {
+    dbg!(request.uri().path());
```

Comment:
This should be removed before merging. (Although perhaps this anticipates adding the check for forbidden words, to make sure it works?!?)

Reply:
Exactly. :)